packages/block-editor/src/components/block-inspector/index.js

```diff
@@ -129,6 +129,12 @@ const BlockInspectorSingleBlock = ( {
 				</div>
 			) }
 			<InspectorControls.Slot bubblesVirtually={ bubblesVirtually } />
+			<InspectorControls.Slot
+				__experimentalGroup="dimensions"
+				bubblesVirtually={ false }
+				header={ __( 'Dimensions' ) }
+				label={ __( 'Dimensions options' ) }
```

Comment:
The `header` is the panel header for the ToolsPanel. The `label` is used for the dropdown menu's label.

As the use of the `ToolsPanel` component becomes more varied, I can see cases where the header may wish to contain HTML markup, e.g. color swatch indicators, or the label for the panel's menu needs to be more than just `<header> options`.

Reply:
Do we have examples of the header of the Tools Panel being more varied design-wise? This might not prove to be a necessary affordance and might be good to tighten it first. (The label on the dropdown could also be better as just "Options" or nothing at all.)

Reply:
> Do we have examples of the header of the Tools Panel being more varied design-wise? 

I mistakenly took it that the color block support panel when switched to the tools panel would continue to display its color swatches in the header as it does now.  Revisiting the designs in https://github.com/WordPress/gutenberg/issues/27331 shows the color panel without swatches. So these props could be merged.

> The label on the dropdown could also be better as just "Options" or nothing at all.

The `DropdownMenu` component readme states the `label` is required for accessibility purposes so if not merging `header` and `label` perhaps your "Options" suggestion better than nothing at all?

Would it be acceptable to make these changes to the `ToolsPanel` component in a separate PR? 

Either way, I can update this tomorrow.

Reply:
Yes, let's merge the props for now and we can see if there's a need to extend it later. Without a collapsed state, there's not much need for previews like the color swatch at the moment.

Reply:
The `header` and `label` props for the `ToolsPanel` have been merged ( https://github.com/WordPress/gutenberg/pull/34157/commits/ad5fb02c6ffec757b5fe2beb9ac026cff06746a9) . I went with `label` as that seems to better reflect how the text is used and the limitations imposed upon it.

✅ ToolsPanel tests are passing
✅ Post editor panel for dimensions support is working
✅ Global styles panel in site editor is okay
✅ Storybook entry for tools panel was updated as well

Thanks for the help refining this. Hopefully, this is getting close to being ready to land now.